client/lib/plugins/utils.js

```diff
@@ -207,42 +204,6 @@ PluginUtils = {
 		} );
 	},
 
-	addWpcomIcon: function( plugin ) {
-		if ( plugin.slug && plugin.wpcom ) {
-			return _assign( {}, plugin, { icon: '/calypso/images/upgrades/plugins/' + plugin.slug + '.png' } );
```

Comment:
We should also remove the [corresponding images](https://github.com/Automattic/wp-calypso/tree/master/public/images/upgrades/plugins).